javascript/linkedList/linkedList/linked-list.js

```diff
@@ -4,92 +4,76 @@ const Node = require("./node");
 
 class LinkedList {
   constructor(value) {
-    this.value = value;
     this.head = null;
   }
 
   isEmpty() {
-    return this.head === null ? true : false;
+    return this.head === null;
   }
 
+  // Add a Node to the front of a linked list
   insert(value) {
-    //insert a node at the beginning of a given LL
```

Comment:
As I was going through, I removed a lot of comments. Typically, the rule of thumb is:

1. Comments should be above the code that it is describing
2. Comments should *only* be used to explain to the reader things that they cannot easily understand from the code itself. In other words, it should be for explaining the "why", additional context and for tricky bits of code. A mechanical explanation of the code itself is typically avoided.

In addition to the above, it's a little harder to move and edit code that comes with a bunch of comments. Occasionally, somebody will edit the code but not the associated comment and it can send them on a wild goose chase for something that is no longer true.